assets/base.css

```diff
@@ -1002,6 +1002,13 @@ summary::-webkit-details-marker {
   }
 }
 
+.grid--collapsed .grid__item {
+  --top-left-radius: var(--border-radius);
+  --top-right-radius: var(--border-radius);
+  --bottom-right-radius: var(--border-radius);
+  --bottom-left-radius: var(--border-radius);
+}
```

Comment:
I could also consider including the equivalent blog and collage selectors in the collapsed grid file as well if that is appealing. The original thought being they should mostly live in those specific files instead of a generic `.grid` overrides file. But the collage code at least has certainly bloated a bit, so that could be a conversation worth having.

Reply:
Yeah 🤔  I think maybe we can do some testing once the functionality and code is approved. To compare the different approaches in terms of where to keep that CSS.